From 96dca4bffea70c91f97922bcd98ba4d306b2eded Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Tue, 16 Jun 2015 14:19:18 +0800
Subject: [PATCH] dts: rockchip: rename property 'i2s-controller' to
 'audio-controller'

because we have i2s controller, spdif controller, so we use audio-controller
to include both.

Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 .../devicetree/bindings/sound/rockchip-audio.txt     |  4 ++--
 arch/arm/boot/dts/rk3036-new.dts                     |  6 +++---
 arch/arm/boot/dts/rk3036-rk88.dts                    |  6 +++---
 arch/arm/boot/dts/rk3036-sdk.dts                     |  6 +++---
 arch/arm/boot/dts/rk3128-box-rk88.dts                |  2 +-
 arch/arm/boot/dts/rk3128-box.dts                     |  2 +-
 arch/arm/boot/dts/rk312x.dtsi                        |  6 +++---
 arch/arm/boot/dts/rk3188-lr097.dts                   |  6 +++---
 arch/arm/boot/dts/rk3188-tb.dts                      |  8 ++++----
 arch/arm/boot/dts/rk3288-chrome.dts                  | 10 +++++-----
 arch/arm/boot/dts/rk3288-p977.dts                    |  6 +++---
 arch/arm/boot/dts/rk3288-p977_8846.dts               |  6 +++---
 arch/arm/boot/dts/rk3288-tb.dts                      | 10 +++++-----
 arch/arm/boot/dts/rk3288-tb_8846.dts                 | 10 +++++-----
 arch/arm/boot/dts/rk3288-tesco.dts                   | 10 +++++-----
 arch/arm64/boot/dts/rk3368-box-r88.dts               | 12 ++++++------
 arch/arm64/boot/dts/rk3368-box.dts                   | 12 ++++++------
 arch/arm64/boot/dts/rk3368-p9_818.dts                |  6 +++---
 arch/arm64/boot/dts/rk3368-tb_8846.dts               | 10 +++++-----
 19 files changed, 69 insertions(+), 69 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/rockchip-audio.txt b/Documentation/devicetree/bindings/sound/rockchip-audio.txt
index 413bfe500d13..55025950dfef 100644
--- a/Documentation/devicetree/bindings/sound/rockchip-audio.txt
+++ b/Documentation/devicetree/bindings/sound/rockchip-audio.txt
@@ -24,7 +24,7 @@ rockchip-audio {
 	dais {
 		dai0 {
 			audio-codec = <&codec>;
-			i2s-controller = <&i2s>;
+			audio-controller = <&i2s>;
 			format = "i2s";
 			//continuous-clock;
 			//bitclock-inversion;
@@ -58,7 +58,7 @@ rockchip-audio {
         dais {
                 dai0 {
                         audio-codec = <&codec>;
-                        i2s-controller = <&i2s1>;
+                        audio-controller = <&i2s1>;
                         format = "i2s";
                         //continuous-clock;
                         //bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3036-new.dts b/arch/arm/boot/dts/rk3036-new.dts
index 8ff6addfcfe6..30ea4737b58b 100755
--- a/arch/arm/boot/dts/rk3036-new.dts
+++ b/arch/arm/boot/dts/rk3036-new.dts
@@ -127,7 +127,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -138,7 +138,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -148,7 +148,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3036-rk88.dts b/arch/arm/boot/dts/rk3036-rk88.dts
index cc2b5f58eb33..e997bcc1f608 100755
--- a/arch/arm/boot/dts/rk3036-rk88.dts
+++ b/arch/arm/boot/dts/rk3036-rk88.dts
@@ -124,7 +124,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -134,7 +134,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -144,7 +144,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3036-sdk.dts b/arch/arm/boot/dts/rk3036-sdk.dts
index 15a598b6abb1..52ecd752509e 100755
--- a/arch/arm/boot/dts/rk3036-sdk.dts
+++ b/arch/arm/boot/dts/rk3036-sdk.dts
@@ -110,7 +110,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -120,7 +120,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -130,7 +130,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3128-box-rk88.dts b/arch/arm/boot/dts/rk3128-box-rk88.dts
index 080336f2e02b..4c265eee23c0 100755
--- a/arch/arm/boot/dts/rk3128-box-rk88.dts
+++ b/arch/arm/boot/dts/rk3128-box-rk88.dts
@@ -76,7 +76,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
diff --git a/arch/arm/boot/dts/rk3128-box.dts b/arch/arm/boot/dts/rk3128-box.dts
index 844e0c473f87..c42731ffe9da 100755
--- a/arch/arm/boot/dts/rk3128-box.dts
+++ b/arch/arm/boot/dts/rk3128-box.dts
@@ -63,7 +63,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 58add870beef..46fb32d7b5c9 100755
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -976,7 +976,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -995,7 +995,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec>;
-				i2s-controller = <&i2s1>;
+				audio-controller = <&i2s1>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -1005,7 +1005,7 @@
 			};
 			dai1 {
 				audio-codec = <&codec>;
-				i2s-controller = <&i2s1>;
+				audio-controller = <&i2s1>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3188-lr097.dts b/arch/arm/boot/dts/rk3188-lr097.dts
index 51af9b197aa5..9ca13b42739d 100644
--- a/arch/arm/boot/dts/rk3188-lr097.dts
+++ b/arch/arm/boot/dts/rk3188-lr097.dts
@@ -71,7 +71,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -87,7 +87,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rk610_codec>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -103,7 +103,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3188-tb.dts b/arch/arm/boot/dts/rk3188-tb.dts
index 2d8c2fc9bf76..ae319eb725c0 100644
--- a/arch/arm/boot/dts/rk3188-tb.dts
+++ b/arch/arm/boot/dts/rk3188-tb.dts
@@ -97,7 +97,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -113,7 +113,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rk610_codec>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -129,7 +129,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -145,7 +145,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
diff --git a/arch/arm/boot/dts/rk3288-chrome.dts b/arch/arm/boot/dts/rk3288-chrome.dts
index 44c622dda6ab..d50420099480 100755
--- a/arch/arm/boot/dts/rk3288-chrome.dts
+++ b/arch/arm/boot/dts/rk3288-chrome.dts
@@ -92,7 +92,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -108,7 +108,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -118,7 +118,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -134,7 +134,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -144,7 +144,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3288-p977.dts b/arch/arm/boot/dts/rk3288-p977.dts
index a6072b0ad3bd..c567b687e2f9 100755
--- a/arch/arm/boot/dts/rk3288-p977.dts
+++ b/arch/arm/boot/dts/rk3288-p977.dts
@@ -108,7 +108,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -124,7 +124,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};		
 	};
@@ -134,7 +134,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3288-p977_8846.dts b/arch/arm/boot/dts/rk3288-p977_8846.dts
index b092288ea96d..c36df0d7c263 100755
--- a/arch/arm/boot/dts/rk3288-p977_8846.dts
+++ b/arch/arm/boot/dts/rk3288-p977_8846.dts
@@ -110,7 +110,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -126,7 +126,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};		
 	};
@@ -136,7 +136,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3288-tb.dts b/arch/arm/boot/dts/rk3288-tb.dts
index 74e89529c61d..71ee1bc56086 100755
--- a/arch/arm/boot/dts/rk3288-tb.dts
+++ b/arch/arm/boot/dts/rk3288-tb.dts
@@ -108,7 +108,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -124,7 +124,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -134,7 +134,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -150,7 +150,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -160,7 +160,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3288-tb_8846.dts b/arch/arm/boot/dts/rk3288-tb_8846.dts
index e32edd8dc136..230a75d66890 100755
--- a/arch/arm/boot/dts/rk3288-tb_8846.dts
+++ b/arch/arm/boot/dts/rk3288-tb_8846.dts
@@ -120,7 +120,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -136,7 +136,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -146,7 +146,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -162,7 +162,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -172,7 +172,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
diff --git a/arch/arm/boot/dts/rk3288-tesco.dts b/arch/arm/boot/dts/rk3288-tesco.dts
index 7abfad5762e0..50c0a14fc5b4 100755
--- a/arch/arm/boot/dts/rk3288-tesco.dts
+++ b/arch/arm/boot/dts/rk3288-tesco.dts
@@ -92,7 +92,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -108,7 +108,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 				format = "spdif";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -124,7 +124,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -140,7 +140,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -150,7 +150,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s>;
+				audio-controller = <&i2s>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
diff --git a/arch/arm64/boot/dts/rk3368-box-r88.dts b/arch/arm64/boot/dts/rk3368-box-r88.dts
index 907f0dffd92f..de019ed08fcb 100755
--- a/arch/arm64/boot/dts/rk3368-box-r88.dts
+++ b/arch/arm64/boot/dts/rk3368-box-r88.dts
@@ -129,7 +129,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -147,7 +147,7 @@
 			dais {
 				dai0 {
 					audio-codec = <&codec_hdmi_spdif>;
-					i2s-controller = <&spdif>;
+					audio-controller = <&spdif>;
 				};
 			};
 		};
@@ -156,7 +156,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rk1000_codec>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 			};
 		};
@@ -166,7 +166,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -182,7 +182,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -192,7 +192,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
diff --git a/arch/arm64/boot/dts/rk3368-box.dts b/arch/arm64/boot/dts/rk3368-box.dts
index a52821ee628c..133e57d3c294 100755
--- a/arch/arm64/boot/dts/rk3368-box.dts
+++ b/arch/arm64/boot/dts/rk3368-box.dts
@@ -129,7 +129,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -147,7 +147,7 @@
 			dais {
 				dai0 {
 					audio-codec = <&codec_hdmi_spdif>;
-					i2s-controller = <&spdif>;
+					audio-controller = <&spdif>;
 				};
 			};
 		};
@@ -156,7 +156,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rk1000_codec>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 			};
 		};
@@ -166,7 +166,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -182,7 +182,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -192,7 +192,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
diff --git a/arch/arm64/boot/dts/rk3368-p9_818.dts b/arch/arm64/boot/dts/rk3368-p9_818.dts
index f3032d557403..8943150c823e 100755
--- a/arch/arm64/boot/dts/rk3368-p9_818.dts
+++ b/arch/arm64/boot/dts/rk3368-p9_818.dts
@@ -111,7 +111,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 			};
 		};
@@ -122,7 +122,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -132,7 +132,7 @@
                 dais {
                         dai0 {
                                 audio-codec = <&es8316>;
-                                i2s-controller = <&i2s0>;
+                                audio-controller = <&i2s0>;
                                 format = "i2s";
                         };
 		};
diff --git a/arch/arm64/boot/dts/rk3368-tb_8846.dts b/arch/arm64/boot/dts/rk3368-tb_8846.dts
index 59b43784404c..13c8692fdba0 100644
--- a/arch/arm64/boot/dts/rk3368-tb_8846.dts
+++ b/arch/arm64/boot/dts/rk3368-tb_8846.dts
@@ -125,7 +125,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_i2s>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -141,7 +141,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&codec_hdmi_spdif>;
-				i2s-controller = <&spdif>;
+				audio-controller = <&spdif>;
 			};
 		};
 	};
@@ -151,7 +151,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt5631>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -167,7 +167,7 @@
 		dais {
 			dai0 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "i2s";
 				//continuous-clock;
 				//bitclock-inversion;
@@ -177,7 +177,7 @@
 			};
 			dai1 {
 				audio-codec = <&rt3261>;
-				i2s-controller = <&i2s0>;
+				audio-controller = <&i2s0>;
 				format = "dsp_a";
 				//continuous-clock;
 				bitclock-inversion;
-- 
2.35.3

